src/components/page-elements/seo/seo.js

```diff
@@ -50,7 +51,7 @@ export const SEO = ({ description, title, no_index }) => {
             htmlAttributes={{
                 lang,
             }}
-            title={title}
+            title={title + ' - Derivcrypto'}
```

Comment:
we can't do it this way?
```suggestion
            title={`${title} - Derivcrypto`}
```